js/ui/main.js

```diff
@@ -916,6 +916,62 @@ function notifyError(msg, details) {
 }
 
 /**
+ * formatLogArgument:
+ * @arg (any): A single argument.
+ * @recursion (int): Keeps track of the number of recursions.
+ * @depth (int): Controls how deeply to inspect object structures.
+ *
+ * Used by _log to handle each argument type and its formatting.
+ */
+function formatLogArgument (arg, recursion=0, depth) {
```

Comment:
Yeah its cjs dependency is out of date. Cinnamon 3.6 will be using cjs depending on mozjs38. You can try [building Cinnamon](https://github.com/linuxmint/mint-dev-tools) instead.